requirements-neuron.txt

```diff
@@ -0,0 +1,15 @@
+ninja  # For faster builds.
+psutil
+ray >= 2.5.1
```

Comment:
Do we use ray for distributed inference on Inferentia/Trainium? Otherwise, skipping it would not raise import errors.

Reply:
For now, we are not using ray for distributed inference on Inferentia/Trainium.

Reply:
removed ray from neuronx